Package/Installer.cs

```diff
@@ -274,7 +274,7 @@ private void WaitForPackageFilesFree(string tapDir, List<string> packagePaths)
 
                 while (isPackageFilesInUse(tapDir, packagePaths, exclude))
                 {
-                    var req = new AbortOrRetryRequest(inUseString) {Response = AbortOrRetryResponse.Abort};
+                    var req = new AbortOrRetryRequest("Package files are in use", inUseString) {Response = AbortOrRetryResponse.Abort};
```

Comment:
```suggestion
                    var req = new AbortOrRetryRequest("Package files are in use.", inUseString) {Response = AbortOrRetryResponse.Abort};
```

I guess we should use either title case or proper punctuation here.